package.json

```diff
@@ -143,19 +148,65 @@
+    "prebuild-desktop": "npm run install-if-needed",
+    "build-desktop": "run-p build-server build-css",
+    "postbuild-desktop": "npm run bundle",
+    "prebuild-server": "mkdirp build",
+    "build-server": "npm run env -- webpack --display-error-details --config webpack.config.node.js",
+    "build-css": "run-p build-css:*",
+    "build-css:style": "node-sass --include-path client assets/stylesheets/style.scss public/style.css && npm run autoprefixer -- public/style.css && rtlcss public/style.css public/style-rtl.css",
+    "build-css:debug": "node-sass --include-path client --source-map \"public/style-debug.css.map\" assets/stylesheets/style.scss public/style-debug.css && npm run autoprefixer -- public/style-debug.css",
+    "build-css:editor": "node-sass --include-path client assets/stylesheets/editor.scss public/editor.css && npm run autoprefixer -- public/editor.css",
+    "build-css:directly": "node-sass --include-path client assets/stylesheets/directly.scss public/directly.css && npm run autoprefixer -- public/directly.css",
+    "autoprefixer": "postcss -r --use autoprefixer --autoprefixer.browsers \"last 2 versions, > 1%, Safari >= 8, iOS >= 8, Firefox ESR, Opera 12.1\"",
+    "build-devdocs:proptypes-index": "npm run env -- node server/devdocs/bin/generate-proptypes-index.js \"client/**/index.js\" \"client/**/index.jsx\" \"client/**/example.jsx\" \"!**/test/**\"",
+    "build-devdocs:components-usage-stats": "npm run env -- node server/devdocs/bin/generate-components-usage-stats.js \"client/**/*.js\" \"client/**/*.jsx\" \"!**/docs/**\" \"!**/test/**\" \"!**/docs-example/**\"",
+    "build-devdocs:index": "npm run env -- node server/devdocs/bin/generate-devdocs-index \"**/*.md\" \".github/**.md\" \"!build/**\" \"!node_modules/**\" \"!public/**\"",
+    "bundle": "npm run env -- node server/bundler/bin/bundler.js",
+    "update-deps": "npm run rm -- node_modules && npm run rm -- npm-shrinkwrap.json && npm install && npm install && npm shrinkwrap --dev",
+    "preshrinkwrap": "npm prune",
+    "postshrinkwrap": "node bin/trim-shrinkwrap.js && touch node_modules",
+    "dashboard": "webpack-dashboard -- npm start",
+    "precommit": "npm run install-if-needed && node bin/pre-commit-hook.js",
+    "prepush": "npm run install-if-needed && node bin/pre-push-hook.js",
+    "translate": "i18n-calypso --format pot --output-file ./calypso-strings.pot -e date \"**/*.js\" \"**/*.jsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/extensions/**\"",
+    "preanalyze-bundles": "cross-env WEBPACK_OUTPUT_JSON=1 CALYPSO_ENV=production npm run build",
+    "analyze-bundles": "webpack-bundle-analyzer stats.json public -p 9898",
+    "build-docker": "docker build -t wp-calypso .",
+    "docker": "docker run -it --name wp-calypso --rm -p 80:3000 -e NODE_ENV='wpcalypso' -e CALYPSO_ENV='wpcalypso' wp-calypso",
+    "rm": "node bin/rm.js",
+    "clean": "run-p clean:*",
+    "clean:public": "npm run rm -- public .css .css.map .js",
```

Comment:
We need to add `.js.map` to the list. Those files are created when executing `make analyze-bundles` atm. I have another patch open where I add this change.